index.js

```diff
@@ -1,39 +1,342 @@
+${input}`);
+      }
+    }
+  } // end while
+  // go to the next room
+  play(global.game.currentRoom);
 }
+
+
+async function classroom() {
+  console.log(global.game.currentRoom.message);
+ 
+  let inRoom = true;
+
+  // write Foyer logic block
+  while (inRoom) {
+    console.log(`Player Status: ${global.game.user.checkStatus()}`);
+    let input = await ask('\n>_');
+    input = input.toLowerCase();
+
+    if (commonCommands(input)) {
+      // do nothing, because common command
+    } else {
+
+      // take the seven days paper
+      if ( input.includes("attend lecture") || input.includes("sit") ) {
+        console.log('You sit down in your normal seat. And Josh begins the lecture about recursion.');
+        global.game.user.changeStatus('Confused and Hungry');
+
+        // drop the seven days paper
+      } else if (input.includes("go") && input.includes("downstairs")) {
+        global.game.moveToRoom('Foyer');
+        inRoom = false;
+
+        // catch-all for unrecognized input
+      } else {
+        console.log(`Sorry I don't know how to ${input}`);
+      }
+    }
+  } // end while
+  // go to the next room
+  play(global.game.currentRoom);
+}
+
+/**** Stand alone objects ****/
+
+let roomNames = {
+  Main: "182 Main Street",
+  Foyer: "182 Main Street - Foyer",
+  Hallway: "Upstairs Hallway",
+  Classroom: "Second floor BCA Classroom"
+};
+
+let roomMessages = {
+  "182 Main Street":
+  `\n${roomNames.Main}:
+    You are standing on Main Street between Church and South Winooski.
+    There is a door here.A keypad sits on the handle.
+    On the door is a handwritten sign. `,
+
+  "182 Main Street - Foyer":
+  `\n${roomNames.Foyer}:
+    You are in a foyer. Or maybe it's an antechamber. Or a vestibule. 
+    Or an entryway. Or an atrium. Or a narthex. 
+    But let's forget all that fancy flatlander vocabulary, and just call it a foyer. In Vermont,
+    this is pronounced "FO-ee-yurr". A copy of Seven Days lies in a corner.`,
+
+  "Upstairs Hallway":
+  `\n${roomNames.Hallway }:
+    You are standing at the top of the stairs.`,
+
+  "Second floor BCA Classroom":
+    `\n${roomNames.Classroom}:
+      You are in the classroom. Josh is standing at the 
+      front waiting of the classroom`
+};
+
+let rooms = {
+  Main: new Room(roomNames.Main, ["welcome sign"]),
+  Foyer: new Room(roomNames.Foyer, ["seven days"]),
+  Hallway: new Room(roomNames.Hallway),
+  Classroom: new Room(roomNames.Classroom)
+};
+
+// helper function compares an input to a an array of key words
+let compareSimilarity = (input, triggerWords) => {
+  let count = 0
+  let inputArr = input.split(' ')
+  triggerWords.forEach(word =>
+    inputArr.forEach(inputWord => {
+      if (word === inputWord) {
+        count++;
+      }
+    })
+  );
+  if (((count / triggerWords.length) * 100) >= 50) {
+    return true;
+  } else {
+    return false;
+  }
+}
+
+// function that checks for the common commands for every room
+function commonCommands(input) {
```

Comment:
Excellent! try and expand this to encompass all possible commands. Then you can get rid of your bespoke room functions. They're not necessarily bad, but this way is better.